apps/sms/views/shared/js/dialog.js

```diff
@@ -117,7 +117,7 @@ var Dialog = function(params) {
     }
 
     document.body.classList.remove('dialog-animating');
-  });
+  }, 500);
```

Comment:
Just picked an arbitrary value that's greater than the current animation time. Too short, and you may cut-off the animation.